app/src/main/kotlin/com/saltedge/authenticator/features/authorizations/common/AuthorizationContentView.kt

```diff
@@ -88,18 +90,55 @@ class AuthorizationContentView : LinearLayout {
 
     private fun setDescription(description: DescriptionData) {
         val htmlContent = description.hasHtmlContent
+        val textContent = description.hasTextContent
+        val extraContent = description.hasExtraContent
+        val paymentContent = description.hasPaymentContent
+
         descriptionTextView?.setVisible(show = !htmlContent)
         descriptionWebView?.setVisible(show = htmlContent)
-        if (htmlContent) {
-            val html = description.html ?: ""
-            val encodedHtml = android.util.Base64.encodeToString(html.toByteArray(), android.util.Base64.NO_PADDING)
-            descriptionWebView?.loadData(encodedHtml, "text/html", "base64")
-        } else {
-            descriptionTextView?.movementMethod = ScrollingMovementMethod()
-            descriptionTextView?.text = description.text ?: ""
+        paymentView?.setVisible(show = paymentContent)
```

Comment:
at one moment only one of the blocks should be shown: type of payment or text of html. Currently, your code can display them all at once.